### ISSUE TITLE
Possible solution for the opened terminal.

### ISSUE THREAD

**Issue by zachvlat** [flathub/dev.lizardbyte.app.Sunshine#162]
The use of this flatpak was cumbersome but since the official didn't work for me I used this. The proble was that i had to manually run this ```
sudo -i PULSE_SERVER=unix:$(pactl info | awk '/Server String/{print$3}') \
  flatpak run dev.lizardbyte.app.Sunshine```
with an opened terminal so i can use it.

I have a clean proposal to this:

1. Install pkexec
2. Make a script ```
#!/bin/sh
PULSE_SERVER=unix:$(pactl info | awk '/Server String/{print $3}') flatpak run dev.lizardbyte.app.Sunshine
``` , place it somewhere and make it executable
3. change the desktop entry's Exec `Exec=pkexec /path/to/sunshine-wrapper.sh`

I don't think this is a great solution but if we make all these steps into a single script file the experience will be far better.

**Comment by ReenigneArcher** [comment]
> but since the official didn't work for me I used this

What do you mean by this? That the upstream flatpak package didn't work or an upstream package like the `deb` didn't work? The upstream flatpak should be almost equivalent to this, although there are some minor differences in the packaging. I am working to bring them in full sync, so it would be good to have a better understanding of this statement.

**Comment by zachvlat** [comment]
> > but since the official didn't work for me I used this
> 
> What do you mean by this? That the upstream flatpak package didn't work or an upstream package like the `deb` didn't work? The upstream flatpak should be almost equivalent to this, although there are some minor differences in the packaging. I am working to bring them in full sync, so it would be good to have a better understanding of this statement.

That the upstream flatpak (stable not nightly) package didn't work. Also the audio doesn't work as the deb one (I did put the audio manually as the guide say). If there are extra permissions needed, they must be known.

**Comment by ReenigneArcher** [comment]
Okay, thanks for the info.

> stable not nightly

Did you happen to test the nightly/beta version (or only the stable)? We're getting close to make a new stable release.

**Comment by zachvlat** [comment]
> Okay, thanks for the info.
> 
> > stable not nightly
> 
> Did you happen to test the nightly/beta version (or only the stable)? We're getting close to make a new stable release.

No I did not. I 'll install it in the next hours if you want.

**Comment by ReenigneArcher** [comment]
That would be nice. Just to get a double check if we're on the right path. Thank you!

**Comment by zachvlat** [comment]
> That would be nice. Just to get a double check if we're on the right path. Thank you!

![image](https://github.com/user-attachments/assets/5076fdd7-4de0-454a-88bb-a0424a777995)

**Comment by ReenigneArcher** [comment]
@zachvlat how did you start sunshine?

**Comment by zachvlat** [comment]
I started as is. Like a normal user would do (I m not a normal user). This is the menu item and its arguments.

![image](https://github.com/user-attachments/assets/9d93665d-ed5e-43e8-8cc5-d25ab6802545)

**Comment by ReenigneArcher** [comment]
https://docs.lizardbyte.dev/projects/sunshine/en/master/md_docs_2getting__started.html#flatpak

**Comment by zachvlat** [comment]
this is not user friendly though... someone who 'll use the flathub store to install it will not see or use it. we have to find a way to seemlesly implement this. thats the exact reason why i use this script.

**Comment by ReenigneArcher** [comment]
There's two desktop files: https://github.com/LizardByte/Sunshine/tree/master/packaging/linux/flatpak

Is flatpak not exposing these to you?

**Comment by zachvlat** [comment]
No. Not at KDE at least. It has only one launcher that doenst work as is.
